### DISCUSSION TITLE
Fail2ban forum

### DISCUSSION THREAD

**Discussion by andrewbernard** [fail2ban/fail2ban#3831] [General]
Is there a fail2ban community forum? Is this it? [Not meaning to sound stupid...] Many apps have fora outside github.

**Comment by sebres** [comment]
Is it not really something like an "official" forum, but one can surely open here one or another discussion, yes.

The known to me another places were:
- IRC irc.libera.chat, channel fail2ban via IRC-client or via web-chat - https://web.libera.chat/
- [stackexchange](https://stackexchange.com/search?q=fail2ban)... please don't use stackoverflow (it's only about software dev), use serverfault, superuser etc instead
- [debianforum.de](https://debianforum.de/forum/search.php?keywords=fail2ban&sr=topics) (german only)
